src/Nest/Document/Multiple/Bulk/BulkOperation/BulkUpdateBody.cs

```diff
@@ -30,5 +30,8 @@ internal class BulkUpdateBody<TDocument, TPartialUpdate>
 
 		[DataMember(Name = "if_primary_term")]
 		internal long? IfPrimaryTerm { get; set; }
+
+		[DataMember(Name = "_source")]
+		internal Union<bool, ISourceFilter> Source { get; set; }
```

Comment:
When `_source` is `true`, the response for an update looks similar to

```json
    {
      "update" : {
        "_index" : "test",
        "_type" : "_doc",
        "_id" : "1",
        "_version" : 2,
        "result" : "updated",
        "_shards" : {
          "total" : 2,
          "successful" : 1,
          "failed" : 0
        },
        "_seq_no" : 3,
        "_primary_term" : 1,
        "get" : {
          "_seq_no" : 3,
          "_primary_term" : 1,
          "found" : true,
          "_source" : {
            "field1" : "value1",
            "field2" : "value2"
          }
        },
        "status" : 200
      }
    }
```

The `get` property should be mapped on `BulkResponseItem` too.

Reply:
Done